### ISSUE TITLE
NaNs in model output, likely caused by arithmetic underflow

### ISSUE THREAD

**Issue by andriuslauraitis** [Waikato/wekaDeeplearning4j#72]
Hello,

I'm getting this error in deep neural network training.  How can i retune the network to avoid this error?

Best regards,
Andrius L.

**Comment by basedrhys** [comment]
Hi,

Unfortunately, we haven't found an easy solution - the things you can try is using a different **learning rate** or **pretrained model**.

Best,
Rhys
